rocAL/include/pipeline/master_graph.h

```diff
@@ -225,6 +228,9 @@ class MasterGraph {
     BoxEncoderGpu *_box_encoder_gpu = nullptr;
 #endif
     TimingDbg _rb_block_if_empty_time, _rb_block_if_full_time;
+    RocalBatchPolicy _last_batch_policy; // Determines the handling of the last batch when the shard size is not divisible by the batch size. Check RocalLastBatchPolicy() enum for possible values
```

Comment:
shard_size is not divisible by batch size

The description seems confusing

Reply:
Make it -> 
number of files in a shard is not divisible by the batch size